src/unread/__tests__/unreadSelectors-test.js

```diff
@@ -13,55 +14,19 @@ import {
   getUnreadStreamsAndTopicsSansMuted,
 } from '../unreadSelectors';
 
-const unreadStreamData = [
-  {
-    stream_id: 0,
-    topic: 'a topic',
-    unread_message_ids: [1, 2, 3],
-  },
-  {
-    stream_id: 0,
-    topic: 'another topic',
-    unread_message_ids: [4, 5],
-  },
-  {
-    stream_id: 2,
-    topic: 'some other topic',
-    unread_message_ids: [6, 7],
-  },
-];
-
-const unreadPmsData = [
-  {
-    sender_id: 0,
-    unread_message_ids: [11, 12],
-  },
-  {
-    sender_id: 2,
-    unread_message_ids: [13, 14, 15],
-  },
-];
-
-const unreadHuddlesData = [
-  {
-    user_ids_string: '1,2,3',
-    unread_message_ids: [21, 22],
-  },
-  {
-    user_ids_string: '1,4,5',
-    unread_message_ids: [23, 24, 25],
-  },
-];
-
-const unreadMentionsData = [1, 2, 3];
+import * as eg from '../../__tests__/lib/exampleData';
+import {
+  initialState,
+  initialState as initialUnreadState,
```

Comment:
Hmm, what's the reason for importing `initialState` twice (once retaining the `initialState` name, once giving it the `initialUnreadState` name)?

Reply:
Hmm, good catch. That is probably a consequence of letting the IDE add the import :wink: I'll reduce to just `initialUnreadState`.

Reply:
... Or maybe just `initialState`. This is after all the *unread* selector tests... plus that's the name used in `unreadModel-test.js`, and using the same name in both will simplify unifying those in the future.

(The main work to do to unify them is to make all these `unreadSelectors` tests well-typed. I started on that right after converting the reducer tests, then realized it wasn't actually needed for what this branch is doing, and paused that in favor of pursuing the rest of the branch.)